Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 name = "ed"
 version = "0.3.0"
-authors = ["Matt Bell <mappum@gmail.com>"]
+authors = ["Turbofish <team@turbofish.org>"]
 edition = "2018"
-description = "Encode and Decode traits"
-license = "MIT"
+description = "A minimalist crate for deterministic binary encodings in Rust."
```

Comment:
Doesn’t make sense to include “in Rust” in this context since that is implied by being a crate.